doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -3031,6 +3031,21 @@ Modem library
 
 The issues in this section are related to :ref:`nrfxlib:nrf_modem`.
 
+.. rst-class:: v2-6-0
+
+TNSW-59157: The :c:func:`nrf_recv` and :c:func:`nrf_recvfrom` functions erroneously set errno to :c:macro:`NRF_EAGAIN` instead of ``0`` when these three conditions are met:
```

Comment:
```suggestion
TNSW-59157: The :c:func:`nrf_recv` and :c:func:`nrf_recvfrom` functions erroneously return ``-1`` and sets ``errno`` to :c:macro:`NRF_EAGAIN` instead of returning ``0`` when these three conditions are met:
```